### PR TITLE
Julia Kozeeva Task 1 https://julia-kozeeva-task1.firebaseapp.com/

### PR DESCRIPTION
**Pull request by juliakozeeva**

### CODE REVIEW

File: task1/package.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+  "name": "task1",
+  "version": "1.0.0",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "scss": "sass public/style/sass/main.scss public/style/css/main.css"
```
Comment (vadeneev) [inline task1/package.json]: ⭐️

File: task1/public/index.html [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Basic</title>
+    <link rel="stylesheet" href="style/css/main.css">
+  </head>
+  <body>
+    <div class="wrapper">
+      <header class="block block_main">
+        <div class="block_main_inner">
+          <div class="header__logo">
```
Comment (vadeneev) [inline task1/public/index.html]: You are using element out of it's block

File: task1/public/index.html [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+            <img src="assets/logo.png" alt="logo">
+          </div>
+          <h1 class="text_pre-line">working
```
Comment (vadeneev) [inline task1/public/index.html]: naming looks strange if it is element - where is its block, if it is atomic class - why it has so strange syntax ?

File: task1/public/index.html [hunk 4]
```diff
@@ -0,0 +1,158 @@
...
+          </p>
+        </div>
+        <a class="header__scroll" href="#block_about">
+          <img src="assets/scroll.svg" alt="scroll">
+        </a>
+      </header>
+        <main>
+          <div class="block block_about" id="block_about">
+            <div class="block__inner">
+              <h2 class="h2">
```
Comment (vadeneev) [inline task1/public/index.html]: not sure that it is good practice to use .h2 for h2 - maybe it is title or small title, we need to find better solution

File: task1/public/style/css/main.css [hunk 5]
```diff
@@ -0,0 +1,357 @@
...
+.block_main_inner {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  height: inherit;
+  margin-top: 100px;
+}
+
+header .header__logo img {
```
Comment (vadeneev) [inline task1/public/style/css/main.css]: it is not acceptable in BEM to use element selector

File: task1/public/style/sass/base/_typography.scss [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
+  color: #fff;
+}
+.h2 {
+  color: #3c3c3c;
+  font-family: Lato;
+  font-size: 23px;
+  font-weight: 700;
+  text-transform: uppercase;
+  letter-spacing: 5.97px;
+  position: absolute;
```
Comment (vadeneev) [inline task1/public/style/sass/base/_typography.scss]: and it is all about typography ?

File: task1/public/style/sass/layout/_grid.scss [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+    border: solid #fff;
+    border-width: 0 2px 2px 2px;
+  }
+}
+
+@media screen and (max-width: 630px) {
+  .block {
+    padding: 50px;
+  }
+  .block__inner {
```
Comment (vadeneev) [inline task1/public/style/sass/layout/_grid.scss]: You is using &_white but not &__inner - why ?

File: task1/public/style/sass/pages/_what-we-do.scss [hunk 8]
```diff
@@ -0,0 +1,12 @@
+.block_what-we-do {
```
Comment (vadeneev) [inline task1/public/style/sass/pages/_what-we-do.scss]: why what-we-do is modifier ? and modifier of what ?

File: task1/public/style/sass/pages/_our-works.scss [hunk 9]
```diff
@@ -0,0 +1,83 @@
...
+  .our-works__list {
+    display: flex;
+    display: -webkit-box;
+    display: -webkit-flex;
+    display: -ms-flexbox;
+    width: 100%;
+    height: 100%;
+    &__item {
+      position: relative;
+      // width: 28.54%;
```
Comment (vadeneev) [inline task1/public/style/sass/pages/_our-works.scss]: stale code ?

File: task1/public/style/sass/pages/_footer.scss [hunk 10]
```diff
@@ -0,0 +1,58 @@
+footer {
+  .block_footer {
+    padding-bottom: 50px;
+    .footer__button {
+      display: flex;
+      justify-content: center;
+      margin-bottom: 50px;
+      &_link {
+        display: inline-flex;
+        // justify-content: center;
```
Comment (vadeneev) [inline task1/public/style/sass/pages/_footer.scss]: stale code ?

File: task1/public/index.html [hunk 11]
```diff
@@ -0,0 +1,179 @@
...
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Basic</title>
+    <link rel="stylesheet" href="style/css/main.css">
+  </head>
+  <body>
+    <div class="wrapper">
+      <header class="header">
+        <img class="header__logo" src="assets/logo.png" alt="Main logo of our company">
+        <h1 class="header__title">working
```
Comment (vovcyan) [inline task1/public/index.html]: Move it to new line

**Pull request merged**